pydevices/HtsDevices/acq435st.py

```diff
@@ -273,13 +273,22 @@ def init(self):
         uut = acq400_hapi.Acq400(self.node.data(), monitor=False)
         uut.s0.set_knob('set_abort', '1')
 
-        if self.ext_clock.length > 0:
-            uut.s0.set_knob('SYS_CLK_FPMUX', 'FPCLK')
-            uut.s0.set_knob('SIG_CLK_MB_FIN', '1000000')
-        else:
-            uut.s0.set_knob('SYS_CLK_FPMUX', 'ZCLK')
-        freq = int(self.freq.data())
-        uut.s1.set_knob('ACQ43X_SAMPLE_RATE', "%d"%freq)
+        trg = self.trig_mode.data()
+
+        if trg == 'hard':
+            trg_dx = 0
```

Comment:
I would probably remove the string formatting on line 290 and instead just say 
`trg_dx = 'd0'` or `= 'd1'`